### PR TITLE
ADD: custom add domain

### PR DESCRIPTION
**Pull request by sihrc**

### CODE REVIEW

File: IndicoIo/IndicoIo.php [hunk 1]
```diff
@@ -318,17 +321,18 @@ function addData($data, $params=array()) {
     }
 
     function predict($data, $params=array()) {
+			$params['domain'] = $params["domain"] || $this->domain;
     	$params['collection'] = $this->name;
     	try {
-    		$data = Image::processImage($data, 144, true);	
+    		$data = Image::processImage($data, 144, true);
```
Comment (madisonmay) [inline IndicoIo/IndicoIo.php]: Resize to 512

File: IndicoIo/IndicoIo.php [hunk 2]
```diff
@@ -318,17 +321,18 @@ function addData($data, $params=array()) {
...
     	} catch (ImageException $e) {}
     	return IndicoIo::_callService($data, 'custom', 'predict', $params);
     }
 
     function removeExample($data, $params=array()) {
-    	$params['collection'] = $this->name; 
+    	$params['collection'] = $this->name;
     	try {
-    		$data = Image::processImage($data, 144, true);	
+    		$data = Image::processImage($data, 144, true);
```
Comment (madisonmay) [inline IndicoIo/IndicoIo.php]: Resize to 512

**Comment by sihrc** [comment]
512 changed in other branch.

**Comment by madisonmay** [comment]
K, great.

**Comment by anneCarlson** [comment]
What's going on with indents here?

**Pull request merged**